crates/red_knot_python_semantic/resources/mdtest/suppressions/type_ignore.md

```diff
@@ -38,16 +38,15 @@ For example, the `type: ignore` comment in this example suppresses the error of
 `"test"` and adding `"other"` to the result of the cast.
 
 ```py
-# fmt: off
 from typing import cast
 
 y = (
-    cast(int, "test" +
-            # TODO: Remove the expected error after implementing `invalid-operator` for binary expressions
-            # error: [unused-ignore-comment]
-            2 # type: ignore
+    # error: [unsupported-operator]
+    cast(
+        int,
+        2 + "test",  # type: ignore
     )
-    + "other"  # TODO: expected-error[invalid-operator]
+    + "other"
```

Comment:
I don't think it does because it no longer asserts that the diagnostic can be suppressed by a suppression comment on the line the expression start/ends (the `2 + "test"` expression now starts/ends at the same line).

Can we keep the binary expression over two lines?

```suggestion
y = (
    # error: [unsupported-operator]
    cast(
        int,
        2 + 
        	"test",  # type: ignore
    )
    + "other"
```

Reply:
What's important (and I think you got that right) is that the inner `type: ignore` shouldn't suppress the error from adding `int` and `"other"`

Reply:
After talking about this, we concluded that the modification here is fine. The other property ("can be suppressed by a suppression comment on the line the expression start/ends") is already tested above.